src/server/controllers/signS3.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
```

Comment:
Why is this called signS3, this is confusing.

I think you should call the files

`assetCreateUrl`, `assetGetUrl`, `assetDeleteUrl`.

Right now the naming is all over the place